tests/unit/api/test_tasks.py

```diff
@@ -17,6 +17,8 @@ def test_get(self, test_client, monkeypatch):
                 "status": True,
                 "task": "add_targets",
                 "last_update": "2023-11-17T09:54:15.762882",
+                "message": None,
```

Comment:
Why there is a new `message` field when there is another `message` attribute under `details`?
I am a little confused.

Reply:
Details can have `Dict[str, Any]`, which could allow an extra message field.
But for clarity in the test, I agree we should remove it.
@fsavoia, I think it is good to remove it or add something else.